src/views/Auction/components/AuctionHeader/index.spec.tsx

```diff
@@ -0,0 +1,72 @@
+// Externals
+
+import React from 'react'
+import { render } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import { BigNumber } from 'ethers'
+import dayjs, { Dayjs } from 'dayjs'
+
+// Component
+import { secondsTohms, AuctionHeader } from './index'
+
+// Utils
+import { convertUtcTimestampToLocal } from 'src/utils/date'
+import { isAuctionOpen, isAuctionUpcoming } from 'src/mesa/auction'
+
+// Icons
+import Omen from 'src/assets/svg/Omen.svg'
+
+describe('seconds to HMS function', () => {
+  describe('convert seconds into different formats', () => {
+    test('tests conversion of seconds to minutes', () => {
+      expect(secondsTohms(60)).toBe('1m ')
+    }),
+      test('tests conversion of seconds into days, hours, minutes', () => {
+        expect(secondsTohms(20000)).toBe('5h 33m 20s')
+      }),
+      test('tests negative input', () => {
+        expect(() => {
+          secondsTohms(-100)
+        }).toThrow('seconds cannot be negative')
+      })
+  })
+})
+
+// // variables
+const addHours = (dayjsInstance: Dayjs, hours: number) => dayjsInstance.clone().add(hours, 'h')
+const utcDate = dayjs(new Date().toUTCString())
+const dateUTC = dayjs.unix(utcDate.unix())
+
+describe('tests AuctionHeader react component', () => {
```

Comment:
`<ComponentName>`  is sufficient in my opinion. Is there a convention for the describe clause?